src/js/actions/transform.js

```diff
@@ -556,10 +575,12 @@ define(function (require, exports) {
      * @param {Layer|Immutable.Iterable.<Layer>} layerSpec Either a Layer reference or array of Layers
      * @param {{w: number=, h: number=}} size
      * @param {string} refPoint reference point, vertical position first then horizontal
+     * @param {object} options Batch play options
      *
      * @returns {Promise}
      */
-    var setSize = function (document, layerSpec, size, refPoint) {
+    var setSize = function (document, layerSpec, size, refPoint, options) {
```

Comment:
Please refer to https://github.com/adobe-photoshop/spaces-design/pull/3421#discussion_r47837901 for my reply.